opendbc/car/honda/fingerprints.py

```diff
@@ -559,6 +559,44 @@
       b'77959-T6A-P110\x00\x00',
     ],
   },
+  CAR.HONDA_ODYSSEY_BOSCH: {
+    (Ecu.transmission, 0x18da1ef1, None): [
+      b'28102-5MX-A100\x00\x00',  # 2021 Touring, 2022 Elite - c443ca9eee127f66
+      b'28102-5MX-A200\x00\x00',  # 2023 (unknown trim), 2024 Touring - 680f1d45486d2971
+      b'28102-5MX-A410\x00\x00',  # 2025 Elite - e0c2250590b16ce0
+      b'28102-5MX-C100\x00\x00',  # 2022 Touring
```

Comment:
Can you remove the fingerprint comments, but put the dongle IDs in a comment in the PR? If there's a firmware version that isn't tied to a known dongle or route, remove it for now. The idea is for all fingerprint additions to be traceable back to a dongle in git/PR history rather than the fingerprints file itself.